instances/molecularEntity/DNQX.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/DNQX",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "DNQX (6,7-dinitroquinoxaline-2,3-dione) is a competitive antagonist at AMPA and kainate receptors, two ionotropic glutamate receptor (iGluR) subfamilies.",
+  "description": "It is used in a variety of molecular biology subfields, notably neurophysiology, to assist researchers in determining the properties of various types of ion channels and their potential applications in medicine.",
```

Comment:
Please add the source of the description. Please adapt so that the description can be read without the definition.